packages/gatsby-theme-store/src/components/HomePage/SEO/useSitelinksSearchBoxJsonLd.ts

```diff
@@ -0,0 +1,31 @@
+import { useLocation } from '@reach/router'
+import type { PageProps } from 'gatsby'
+
+type Options = PageProps<unknown>
+
+interface Return {
+  '@context': 'https://schema.org'
+  '@type': 'WebSite'
+  url: string
+  potentialAction: {
+    '@type': 'SearchAction'
+    target: string
+    'query-input': string
+  }
+}
+
+export const useSiteLinksSearchBoxJsonLd = (_: Options): Return | null => {
```

Comment:
What this `Ld` means?

Reply:
Json-ld is a format used for generating structured data. You can take a look in [here](https://www.searchenginejournal.com/upgrade-to-json-ld-structured-data/319327/#:~:text=JSON%2DLD%20structured%20data%20is,near%20the%20closing%20body%20tag.)

Reply:
Hmm, nice, i didn't know that this exists